src/nodeDebugAdapter.ts

```diff
@@ -457,23 +457,26 @@ export class NodeDebugAdapter extends ChromeDebugAdapter {
         }
     }
 
-    public configurationDone(): Promise<void> {
+    public async configurationDone(): Promise<void> {
         if (!this.chrome) {
             // It's possible to get this request after we've detached, see #21973
-            return super.configurationDone();
+            await super.configurationDone();
+            return;
```

Comment:
This is equivalent to `return super.configurationDone()`, you could leave it that way for simplicity

Reply:
Changed